src/Microsoft.Health.Dicom.Core/Features/Validation/ValidationErrorCode.cs

```diff
@@ -0,0 +1,84 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Validation
+{
+    /// <summary>
```

Comment:
This is quite a bit for the `<summary/>`. Remember that this content is part of intellisense and may be rendered as part of docs. So I'd keep the summary to be the brief description of what it is. Details can be structured inside of `<remarks/>` and any notes for our own devs can be standard code comments like `//` or `/* ... */`